app/concepcts/reservation_seats/reservation_seat_repository.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ReservationSeats
+  class ReservationSeatRepository
```

Comment:
Remember that if something is in namespace `ReservationSeats`, you don't have to include this name in class itself. So instead of having duplicate - `ReservationSeats::ReservationSeatRepository`, you can create simple `ReservationSeats::Repository`